source/priority_queue.c

```diff
@@ -198,12 +222,33 @@ bool aws_priority_queue_is_valid(const struct aws_priority_queue *const queue) {
     bool backpointer_list_item_size = queue->backpointers.item_size == sizeof(struct aws_priority_queue_node *);
     bool lists_equal_lengths = queue->backpointers.length == queue->container.length;
     bool backpointers_non_zero_current_size = queue->backpointers.current_size > 0;
+
+    /* This check must be guarded, as it is not efficient, neither
+     * when running tests nor CBMC */
+#ifdef PQ_BACKPOINTERS_CHECK
```

Comment:
Is this the best name here?

Reply:
I changed that name to `AWS_DEEP_CHECKS`, which should be used to guard any validity check that is not O(1).